README.md

```diff
@@ -27,3 +27,22 @@ then navigate to [localhost:4000](http://localhost:4000)
 Copyright (C) 2014-2016 Alexander Petrov, Michael S. Klishin, Zack Maril, and the ClojureWerkz team.
 
 Distributed under the Eclipse Public License, the same as Clojure.
+
+### Using alert tags
+
+There are four types of alert highlighting to inform users: note, tip, warning, and important. You can insert an alert by using any of the following code in a markdown file. 
+
+{% include note.html content="This is a note." %}
+
+{% include tip.html content="This is a tip." %}
+
+![alert example](/assets/images/note_tip_alerts.png)
+
+To include new paragraphs, just add the `<br/>` tag within the content, like this:
+
+{% include warning.html content="This is a warning. `<br/>` This is the second line of the warning." %}
+
+{% include important.html content="This is for an important message." %}
+
+![alert example2](/assets/images/warning_important_alerts.png)
+
```

Comment:
I added the images to show what the alerts looked like. They're definitely not necessary and I can remove them if it just doesn't make any sense to have them there.

> On Apr 27, 2016, at 13:16, Larsson Omberg <notifications@github.com> wrote:
> 
> In README.md:
> 
> > +There are four types of alert highlighting to inform users: note, tip, warning, and important. You can insert an alert by using any of the following code in a markdown file. 
> > +
> > +{% include note.html content="This is a note." %}
> > +
> > +{% include tip.html content="This is a tip." %}
> > +
> > +![alert example](/assets/images/note_tip_alerts.png)
> > +
> > +To include new paragraphs, just add the `<br/>` tag within the content, like this:
> > +
> > +{% include warning.html content="This is a warning. `<br/>` This is the second line of the warning." %}
> > +
> > +{% include important.html content="This is for an important message." %}
> > +
> > +![alert example2](/assets/images/warning_important_alerts.png)
> > +
> What is the purpose of including the images? Are these necessary for the static site or only for this readme?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>